web/src/features/map-controls/MapButton.tsx

```diff
@@ -9,35 +8,29 @@ interface MapButtonProperties {
   tooltipText?: string;
   className?: string;
   dataTestId?: string;
-  asToggle?: boolean;
   ariaLabel?: string;
+  backgroundClasses?: string;
 }
 
 export default function MapButton({
```

Comment:
We now have a button within a button, which messes up keyboard navigation:

![Screenshot 2024-07-16 at 15 04 43](https://github.com/user-attachments/assets/c5f943a4-882b-4d3e-a2de-1f80b580e814)

https://github.com/user-attachments/assets/53441915-adf5-4244-82bf-5e2e0fa8b4bb